spec/tck/src/main/java/io/smallrye/asyncapi/tck/StreetlightsAppTest.java

```diff
@@ -109,14 +109,13 @@ public void testChannels(String type) {
         ValidatableResponse vr = callEndpoint(type);
 
         String topic = "'smartylighting/streetlights/1/0/action/{streetlightId}/dim'";
-        vr.body("channels." + topic + ".channel", equalTo(topic.replace("\'", "")));
+        vr.body("channels." + topic + ".channel", equalTo(null));
```

Comment:
please use other matcher instead of _equal(null)_

Reply:
test was unnecessary